setup.py

```diff
@@ -1,6 +1,45 @@
 from setuptools import setup, find_packages
 
+# read the contents of your README file
+from pathlib import Path
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
+
 setup(
-    name='nowcasting_dataset',
-    version='0.1',
-    packages=find_packages())
+    name="nowcasting_dataset",
+    version="0.1.2",
+    license="MIT",
+    description="Nowcasting Dataset",
+    author="Jack Kelly, Peter Dudfield, Jacob Bieker",
+    author_email="info@openclimatefix.org",
+    company="Open Climate Fix Ltd",
+    install_requires=[
```

Comment:
should this be from a requirements file?

Reply:
Yes, it sometimes doesn't seem to work if I do it that way in Actions, and complains about the txt not existing, but I can change it back to it!

Reply:
Okay, turns out including a `MANIFEST.in` is the way to fix it.